### PR TITLE
Minor uialignment bugfixes

### PR DESCRIPTION
**Pull request by vishwam**
This PR contains various bugfixes discovered when updating iotcentral to the uialignment branch

### CODE REVIEW

File: lib/components/Masthead/Masthead.module.scss [hunk 1]
```diff
@@ -6,7 +6,7 @@ $masthead-branding-width: 50%;
 $masthead-brandname-width: $grid-size * 50;
 $masthead-application-breakpoint: 3 * $masthead-brandname-width - 1;
 $masthead-collapse-start: 125 * $grid-size;
-$thumbnail-size: $icon-size-base * 2;
+$thumbnail-size: 36px; // match @user-thumbnail-size in Thumbnail.module.scss
```
Comment (vincenzocaruso) [inline lib/components/Masthead/Masthead.module.scss]: nit: $grid-size

File: lib/components/SearchInput/SearchInput.tsx [hunk 2]
```diff
@@ -23,24 +22,18 @@ export interface SearchInputProps extends React.Props<TextInputType> {
...
+                ? <ActionTriggerButton onClick={this.props.onSubmit} icon='forward' className={css('search-button')} attr={{ 
+                    button: { title: this.props.label, type: 'submit' },
+                    container: this.props.attr ? this.props.attr.postfix : null 
+                }} />
                 : null
         );
 
         return (
-            <div className={css('search-input-container', this.props.containerClassName)}>
+            <form className={css('search-input-container', this.props.containerClassName)} onSubmit={this.props.onSubmit}>
```
Comment (vincenzocaruso) [inline lib/components/SearchInput/SearchInput.tsx]: role='search'

File: lib/components/Masthead/Masthead.module.scss [hunk 3]
```diff
@@ -245,11 +245,12 @@ $search-max-size: 100 * $grid-size;
...
 
                 &.user-btn {
-                    >:global(.icon-alias-user) {
-                        width: $thumbnail-size;
-                        height: $thumbnail-size;
-                        font-size: $font-size-h5;
-                        border-radius: 50%;
+                    width: $thumbnail-size;
+                    height: $thumbnail-size;
+                    border-radius: 100%;
```
Comment (Coord) [inline lib/components/Masthead/Masthead.module.scss]: nit: 50% border-radius, with 100% the browser has to do additional scaling to get the circle

File: lib/components/Masthead/Masthead.module.scss (hunk 1 above)
Comment (vishwam) [reply lib/components/Masthead/Masthead.module.scss]: done

File: lib/components/SearchInput/SearchInput.tsx (hunk 2 above)
Comment (vishwam) [reply lib/components/SearchInput/SearchInput.tsx]: done

File: lib/components/Masthead/Masthead.module.scss (hunk 3 above)
Comment (vishwam) [reply lib/components/Masthead/Masthead.module.scss]: done

**Pull request merged**